emeval/input/phone_view.py

```diff
@@ -167,7 +172,7 @@ def transitions_to_ranges(transition_list, start_tt, end_tt, start_ti, end_ti,
         range_list = []
         range_count_map = dict.fromkeys(unique_trip_ids, 0)
         for (s, e) in zip(start_transitions, end_transitions):
-            # print("------------------------------------- \n %s -> \n %s" % (s, e))
+            print("------------------------------------- \n %s -> \n %s" % (s, e))
```

Comment:
```suggestion
            # print("------------------------------------- \n %s -> \n %s" % (s, e))
```

No need to add to log spew unless we are specifically debugging this